_pages/schedule.html

```diff
@@ -8,7 +8,9 @@
 title: Full Conference Schedule
 ---
 
-<nav class="schedule-nav">
+The 2022 conference schedule is coming soon!
+
+<!-- <nav class="schedule-nav">
```

Comment:
FYI we typically use the `{% comment %}<some code>{% endcomment %}` form so that it doesn't get rendered in the HTML at all.
The changes are fine, though, so if you don't mind making that switch, we'll be golden.

Reply:
Cool, I will make the change

Reply:
Let me know if the current version won't work and if I need to comment each section by itself